gce/scripts/startup-script.sh

```diff
@@ -19,26 +19,31 @@ set -v
 
 # Talk to the metadata server to get the project id
 PROJECTID=$(curl -s "http://metadata.google.internal/computeMetadata/v1/project/project-id" -H "Metadata-Flavor: Google")
-BUCKET=$(curl -s "http://metadata.google.internal/computeMetadata/v1/instance/attributes/BUCKET" -H "Metadata-Flavor: Google")
 
-echo "Project ID: ${PROJECTID}  Bucket: ${BUCKET}"
-
-# get our file(s)
-gsutil cp "gs://${BUCKET}/gce/"** .
+echo "Project ID: ${PROJECTID}"
 
 # Install dependencies from apt
-apt-get update
-apt-get install -yq openjdk-11-jre-headless
+apt-get install -yq openjdk-8-jdk git
```

Comment:
The original is using Java11 - is it intentional to switch to Java8?

Reply:
The original-original is using Java 8. I switched it to Java 11 when troubleshooting to try to get it to work, but turned out that wasn't a factor. I can use 11 though.